views/includes/check-your-answers/trusts.html

```diff
@@ -18,8 +18,8 @@ <h3 class="govuk-heading-m">{{ trust_heading }}</h3>
   {% if isRegistration  %}
     {% set changeLinkUrl = OE_CONFIGS.TRUST_DETAILS_URL %}
   {% else %}
-    {% set stillInvolved = trust.trust_still_involved_in_overseas_entity %}
-    {% set isNotStillInvolved = stillInvolved === "No" %}    
+    {% set trustStillInvolved = trust.trust_still_involved_in_overseas_entity %}
+    {% set isNottrustStillInvolved = trustStillInvolved === "No" %}    
```

Comment:
isNottrustStillInvolved needs a capital T on trust

Reply:
Changed the name slightly while adding this change.